bundles/org.openhab.core/src/main/resources/OH-INF/config/addons.xml

```diff
@@ -38,7 +38,8 @@
 		<parameter name="suggestionFinderSddp" type="boolean">
 			<advanced>true</advanced>
 			<label>SDDP Suggestion Finder</label>
-			<description>Use SDDP network scan to suggest add-ons. Enabling/disabling may take up to 1 minute.</description>
+			<description>Use Control4's Simple Device Discovery Protocol (SDDP) network scan to suggest add-ons.
```

Comment:
This seems like too many words to add when compared to all the other finders' very concise  descriptions. How about just `Control4 SDDP`?
```suggestion
			<description>Use Control4 SDDP network scan to suggest add-ons.
```
Current finder descriptions for reference:
<img width="727" height="535" alt="image" src="https://github.com/user-attachments/assets/9928b7b1-45ed-4d3a-a8d1-7326152e0bcf" />